src/components/Sum.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+export const Sum = ({ a = 0, b = 0 }) => {
+  const text = `Sum of ${a} and ${b} is ${a + b}`;
+
+  return (
+    <p>{text}</p>
+  );
+};
+
+Sum.propsTypes = {
+  a: PropsTypes.number,
+  b: PropsTypes.number,
+};
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
I guess you don't really need this, if you have this 
![image](https://user-images.githubusercontent.com/99331793/176869250-e4ce0cad-2afe-46ae-ba11-398d3521a847.png)